sources/packages/web/src/constants/ecert-constants.ts

```diff
@@ -0,0 +1,34 @@
+import { ECertFailedValidation } from "@/types";
+
+export interface EcertFailedValidationDetail {
+  failedType: ECertFailedValidation;
+  failedMessage: string;
+}
+
+export const ECERT_FAILED_MESSAGES: EcertFailedValidationDetail[] = [
+  {
+    failedType: ECertFailedValidation.InvalidSIN,
+    failedMessage:
+      "Your SIN is invalid and your funding cannot be issued. Contact StudentAid BC for assistance.",
+  },
+  {
+    failedType: ECertFailedValidation.MSFAANotValid,
```

Comment:
We can ask business to have two different messages for the MSFAA since we have the specific error already defined.